docs/script-catalog/person_authentication/person-authentication.md

```
@@ -119,72 +126,76 @@ The `acr` parameter can be configured in the following ways :
+
+An Authentication method is offered by the AS if its ACR value i.e. its
 corresponding custom script is `enabled`.
 
-By default, users will get the default authentication mechanism as 
-specified above. However, **using the OpenID Connect acr_values parameter, 
+By default, users will get the default authentication mechanism as
+specified above. However, **using the OpenID Connect acr_values parameter,
 web and mobile clients can request any enabled authentication mechanism**.
 
 1. Obtain the json contents of a custom script by using a jans-cli command like `get-config-scripts-by-type`, `get-config-scripts-by-inum` etc.
 	Example :
```

markdownlint-cli2 (0.18.1)

Hard tabs
Column: 1

(MD010, no-hard-tabs)